src/toxcore/dht/server/mod.rs

```diff
@@ -746,6 +748,24 @@ impl Server {
         }
     }
 
+    /// Update returned socket address and time of receiving packet
+    fn update_returned_sock(&self, node: &PackedNode, packet_pk: &PublicKey, close_nodes: &mut Kbucket, friends: &mut Vec<DhtFriend>) {
```

Comment:
`update_returned_sock` -> `update_returned_addr` since we update address and network socket has different meaning.